app/models/account.rb

```diff
@@ -108,7 +101,22 @@ def self.get_key(user)
     user.uuid || "sec:#{user.sec_id}"
   end
 
-  private_class_method :account_attrs_from_user, :get_key
+  # Sort the given list of Accounts so the ones with matching ID.me UUID values
+  # come first in the array, this will provide users with a more consistent
+  # experience in the case they have multiple credentials to login with
+  # https://github.com/department-of-veterans-affairs/va.gov-team/issues/6702
+  #
+  # @return [Array]
+  def self.sort_with_idme_uuid_priority(accts)
+    if accts.length > 1
+      data = accts.map(&:attributes)
+      log_message_to_sentry('multiple Account records with matching ids', 'warning', data)
```

Comment:
`data` has PII that we don't want logged to sentry.

Reply:
thanks @annaswims I just made this change to only log the Account ID.  https://github.com/department-of-veterans-affairs/vets-api/pull/4013/commits/43d912c04f27a1643bd66d1242d5a756bef9374b